declarations_site/catalog/static/js/main.js

```diff
@@ -58,6 +58,28 @@ $(function() {
         $(this).toggleClass('open');
     });
 
+    $('.bi-toggle-search a').on('click', function(e){
+        e.preventDefault();
+        console.log(e.target);
+        $('.nav-search').toggleClass('open');
+    });
+
+    $('.bi-analytics-page #cta').on('click', function(e){
```

Comment:
Why do you need .bi-analytics-page part here, if you are addressing the button by id?